pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -90,7 +92,9 @@ func (q *graceTerminateRSList) flushList(handler func(rsToDelete *listItem) (boo
 		}
 		if deleted {
 			klog.InfoS("Removed real server from graceful delete real server list", "realServer", name)
-			q.remove(rs)
```

Comment:
Can we just delete this function now or are there other calls to q.remove now?

Reply:
can't delete `q.remove()`, there other still calls to `q.remove()`, for example:
https://github.com/kubernetes/kubernetes/blob/7d9a6d1de6552f66ca53ee69e072f604b4368c85/pkg/proxy/ipvs/graceful_termination.go#L200